docs/userguide/networking/get-started-overlay.md

```diff
@@ -16,7 +16,6 @@ network. Docker Engine supports multi-host networking out-of-the-box through the
 `overlay` network driver.  Unlike `bridge` networks, overlay networks require
 some pre-existing conditions before you can create one. These conditions are:
 
-* A host with a 3.16 kernel version or higher.
```

Comment:
but there is the kernel version in the commit msg: `Support for overlay network driver in 3.10+ kernels`